Reports/reportModels.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require('mongoose');
```

Comment:
@jmtaber129, normally I would try to separate the database specific items out of the model (schema) files.

Is there a reason you did this here?